cockatrice/src/deck_loader.h

```diff
@@ -28,6 +28,9 @@ class DeckLoader : public DeckList {
     bool loadFromFile(const QString &fileName, FileFormat fmt);
     bool loadFromRemote(const QString &nativeString, int remoteDeckId);
     bool saveToFile(const QString &fileName, FileFormat fmt);
+
+    // overload
+    bool saveToStream_Plain(QTextStream &stream);
```

Comment:
Is `stream` an out param here? We should document that in the header here. CLion picks up javadoc-style comments.